src/index.ts

```diff
@@ -87,4 +88,20 @@ void (async () => {
 			error(String(e));
 		}
 	}
+
+	// If configdir is passed call than add its value to the load operation
+	if (args['confDir']) {
+		const configPath = args['confDir'];
+		try {
+			await load(configPath);
```

Comment:
The load function returns the config object and you are not using it anywhere.

Reply:
I comment below it that the todo list didn't mention what to do with the new config object
okay what i have to do with the new config object ?

Reply:
may I have to deploy package but with the config path instead of the root path?